app/models/camp.rb

```diff
@@ -119,6 +122,11 @@ class Camp < ActiveRecord::Base
     where(is_public: flag)
   }
 
+  scope :is_cocreation, lambda { |flag|
```

Comment:
I believe that the scope is how to use filterrific - so it will take it into account. Not sure on how to do it otherwise.
it looks better the way you write it.
The "" means an empty string. sometimes the value is not. other times it is an empty strings.